indigenous-acknowledgement.md

```diff
@@ -0,0 +1,39 @@
+# Linux Australia's Indigenous Acknowledgement
+
+## Overview
+
+Linux Australia recognises Australia's Indigenous heritage, and the history of inequality and under-representation faced by Indigenous Australians in all sectors of society, including the open source community. Linux Australia, in alignment with its core values [0], seeks to ensure the recognition and representation of Indigenous people and culture throughout its activities, events and administration mechanisms.
```

Comment:
My knowledge of markdown is a tad rusty - does Markdown allow for hyperlinks (where the [0] is)?